cves/CVE-2017-14056.yml

```diff
@@ -324,8 +356,8 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: Video files are complex inputs that must be handled carefully. 
```

Comment:
Here you could go more into detail about what makes video files complex and other common issues with these types of files